src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -54,43 +54,187 @@ limitations under the License.
+          Role/Privilege:   V(iew)
+          Modifier:         F(abric-scoped)
+
+      Other notes:
+        Updated attribute description, as per the spec.
+    -->
+    <attribute side="server" code="0x0001" define="FABRICS" type="ARRAY" entryType="FabricDescriptor" length="320" writable="false" optional="false">
+      <description>Fabrics</description>
+      <access op="read" privilege="view" modifier="fabric-scoped"/>
+    </attribute>
+
+    <!--     
+      Definition:
+      | Id | Name                | Type                                      | Constraint            | Quality | Default | Access | Conformance
+      | 2  | SupportedFabrics    | uint8                                     | 5 to 255              | F       |         | R V    | M
+
+      <access/> Design Rationale:
+        Per Definition:
+          Operation:        R(ead) only. I.e. Write is inferred as not allowed.
+          Role/Privilege:   V(iew)
+          Modifier:         None
+    -->    
+    <attribute side="server" code="0x0002" define="SUPPORTED_FABRICS" type="INT8U" writable="false" optional="false">
+      <description>SupportedFabrics</description>
+      <access op="read" privilege="view"/>
+    </attribute>
+
+    <!--     
+      Definition:
+      | Id | Name                | Type                                      | Constraint            | Quality | Default | Access | Conformance
+      | 3  | CommissionedFabrics | uint8                                     | max SupportedFabrics  | N       |         | R V    | M
+
+      <access/> Design Rationale:
+        Per Definition:
+          Operation:        R(ead) only. I.e. Write is inferred as not allowed.
+          Role/Privilege:   V(iew)
+          Modifier:         None
+    -->    
+    <attribute side="server" code="0x0003" define="COMMISSIONED_FABRICS" type="INT8U" writable="false" optional="false">
+      <description>CommissionedFabrics</description>
+      <access op="read" privilege="view"/>
+    </attribute>
+
+    <!--     
+      Definition:
+      | Id | Name                | Type                                      | Constraint            | Quality | Default | Access | Conformance
+      | 4  | TrustedRootCertificates  | <<list-def>>[<<octstr-def>>]         | max SupportedFabrics  | N       |         | R V    | M
+
+      <access/> Design Rationale:
+        Per Definition:
+          Operation:        R(ead) only. I.e. Write is inferred as not allowed.
+          Role/Privilege:   V(iew)
+          Modifier:         None
+    -->    
     <!-- 400 = 400 bytes for root cert -->
-    <attribute side="server" code="0x0004" define="TRUSTED_ROOTS" type="ARRAY" entryType="OCTET_STRING" length="400" writable="false" optional="false">TrustedRootCertificates</attribute>
+    <attribute side="server" code="0x0004" define="TRUSTED_ROOTS" type="ARRAY" entryType="OCTET_STRING" length="400" writable="false" optional="false">
```

Comment:
Each ENTRY can be 400. Will 400 limit be a problem, or is it ignored?

@bzbarsky-apple

Reply:
At this point, I believe the `length` is completely ignored; it used to be used for allocating space in the attr store, but we don't do that anymore.

Just looking through this quickly, in the ZAP repo we have:
```
% git grep "\.\$\.length"
src-electron/zcl/zcl-loader-silabs.js:        maxLength: attribute.$.length ? attribute.$.length : null,
src-electron/zcl/zcl-loader-silabs.js:        maxLength: item.$.length ? item.$.length : null,
```
and then that's used in `typeSizeAttribute`.  `typeSizeAttribute` is used in `helper-endpointconfig.js` and `helper-tokens.js`, neither of which is relevant to list attributes in Matter.  And in Matter we use `maxLength` directly in `Accessors-src.zapt`, which is also not relevant to list attributes.

Not sure whether we should just drop it from our XML or what.  @vivien-apple any opinions?